VAMobile/src/api/contactInformation/getContactInformation.tsx

```diff
@@ -0,0 +1,37 @@
+import { useQuery } from '@tanstack/react-query'
+
+import { ContactInformationPayload, UserContactInformation } from 'api/types/ContactInformation'
+import { contactInformationKeys } from './queryKeys'
+import { get } from 'store/api'
+import { getFormattedPhoneNumber } from 'utils/common'
+
+/**
+ * Fetch user contact information
+ */
+const getContactInformation = async (): Promise<UserContactInformation | undefined> => {
+  const response = await get<ContactInformationPayload>('/v0/user/contact-info')
+  const contactInformation = response?.data.attributes
+
+  if (contactInformation) {
+    return {
+      ...contactInformation,
+      formattedHomePhone: contactInformation.homePhone && getFormattedPhoneNumber(contactInformation.homePhone),
+      formattedMobilePhone: contactInformation.mobilePhone && getFormattedPhoneNumber(contactInformation.mobilePhone),
+      formattedWorkPhone: contactInformation.workPhone && getFormattedPhoneNumber(contactInformation.workPhone),
+    }
+  }
+}
+
+/**
+ * Returns a query for user contact information
+ */
+export const useContactInformation = () => {
+  return useQuery({
+    queryKey: contactInformationKeys.contactInformation,
+    queryFn: () => getContactInformation(),
+    staleTime: 5000,
```

Comment:
I adjusted the staleTime to 5 seconds because the `ContactInformationScreen` renders the `AddressSummary` component, and both components use this query, which results in 2 calls being made to the contact info endpoint. 5 seconds is enough time to prevent the query from refetching in the child component, and also gives the user some time to select what info they want to edit, since the editing screens also use the query, and would refetch the data.

I manually set the stale time in this query for now, but this touches on an important issue on how we want to manage stale time for our queries, because we can also set a global stale time in the query client, and update the stale time for individual queries when needed. React Query's default stale time of 0 may be a bit aggressive for our app